src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -640,6 +642,49 @@ public void getUsersProfilePictureByProfileId(String profileId, Map<String, Stri
       });
   }
 
+  @Override
+  @Stream
+  public void putUsersProfilePictureByProfileId(String profileId, InputStream entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```

Comment:
@SinghAdes and @Vignesh-kalyanasundaram  Please add logs to this method. I think logging guidelines should be adhered to all the PR for profile pictures stories.